include/veriblock/entities/address_entity.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef ALT_INTEGRATION_ADDRESS_ENTITY_HPP
+#define ALT_INTEGRATION_ADDRESS_ENTITY_HPP
+
+#include <cstdint>
+#include <string>
+
+#include "veriblock/consts.hpp"
+
+namespace VeriBlock {
+
+struct AddressEntity {
```

Comment:
why renamed to AddressEntity?

and why removed existing serialization/deserialization functions?

Reply:
I want to separate data structures and associated parse/serialize/validate functions. We either move everything to address.hpp or move everything out of there. I've chosen the latter.